src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Razor.Sdk.CurrentVersion.targets

```diff
@@ -373,6 +381,11 @@
 
   </Target>
 
+  <Target Name="_EnsureRazorCompilerReferenced" Condition="'$(IsRazorCompilerReferenced)' != 'true'">
+    <Error Text="A reference to Microsoft.AspNetCore.Razor.Design is required for this target to succeed. See todo:fwlink for instructions on how to resolve this." />
```

Comment:
I don't want to block on this, but it would be cool to talk to @vijayrkn or someone else with more SDK experience about how we should make this actionable. Can you log an issue to do a polish pass on this error?